tst/EnergyPlus/unit/Psychrometrics.unit.cc

```diff
@@ -45,15 +45,150 @@
 // OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 // POSSIBILITY OF SUCH DAMAGE.
 
-// EnergyPlus::Pumps Unit Tests
-
 // Google Test Headers
 #include <gtest/gtest.h>
 
+// EnergyPlus Headers
+#include <EnergyPlus/Psychrometrics.hh>
+
 #include "Fixtures/EnergyPlusFixture.hh"
-#include <Psychrometrics.hh>
+using namespace EnergyPlus;
+using namespace EnergyPlus::Psychrometrics;
+
+TEST_F(EnergyPlusFixture, Psychrometrics_PsyTsatFnHPb_Test)
+{
+
+    InitializePsychRoutines();
+
+    // Test 1: Label70 - TEMP. IS FROM  20 C  TO   40 C
+    Real64 H = 7.5223e4;
+    Real64 PB = 1.0133e5;
+    Real64 result = PsyTsatFnHPb_raw(H, PB);
+    Real64 actual_result = F6((H + 1.78637e4), -1.82124e1, 8.31683e-4, -6.16461e-9, 3.06411e-14, -8.60964e-20, 1.03003e-25);
```

Comment:
So with this unit test, we now have the calculations duplicated directly from the function.  This isn't ideal.  All this verifies is that you copy/pasted the evaluations properly into the unit test, not that the value is actually correct.  It would be better to just grab a psych chart and sample a few values and use that independent resource to verify the  functions are producing accurate psychrometric outputs.